src/main/cpp/CMakeLists.txt

```diff
@@ -196,6 +196,7 @@ add_library(
   src/HashJni.cpp
   src/HistogramJni.cpp
   src/HostTableJni.cpp
+  src/HLLPPJni.cpp
```

Comment:
Let's try to be generic.
```suggestion
  src/AggregationJni.cpp
```

Reply:
Renamed to: HLLPPHostUDFJni
AggregationJni is too generic